src/almanack/LoC_tracker.py

```diff
@@ -0,0 +1,26 @@
+"""
+This module calculates the absolute value of lines of code (LoC) changed (added or removed)
+in the given Git repository.
+"""
+
+import git
+
+
+def calculate_loc_changes(repo_path: str):
+    """
+    Finds the total number of code lines changed
+
+    Args:
+        repo_path(str): The path to the git repository
+    Returns:
+        int: Total number of lines added or removed
+    """
+    repo = git.Repo(repo_path)
+
+    total_lines_changed = 0
```

Comment:
Consider the use of a [list comprehension](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions) to combine this line with the for loop to make the necessary calculations (this might depend on how you approach things given the below comment as well).